examples/delete_projects.py

```diff
@@ -0,0 +1,42 @@
+'''
+Created on Nov 8, 2019
+
+@author: ylei
+
+Delete multiple projects and their scans. The project names to be deleted are kept in clean_project.json
+'''
+
+import argparse
+import csv
+import logging
+import sys
+import json
+import arrow
+
+from blackduck.HubRestApi import HubInstance
+
+parser = argparse.ArgumentParser("A program that will delete projects along with their scans")
+#parser.add_argument("project", help="Project name")
+parser.add_argument("-f", "--filename", default="clean_project.json", help="File to keep Project names")
+parser.add_argument("-k", "--keep_scans", action = 'store_true', default=False, help="Use this option if you want to keep scans associated with the project-versions. Default is False, scans will be deleted.")
+parser.add_argument("-b", "--backup_scans", action = 'store_true', default=False, help="Use this option if you want to backup scans associated with the project-versions. Default is False, scans will not be backuped.")
+args = parser.parse_args()
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+logging.getLogger("requests").setLevel(logging.WARNING)
+logging.getLogger("urllib3").setLevel(logging.WARNING)
+
+hub = HubInstance()
+
+filename = args.filename
+with open(filename, 'r+') as f:
+    d = json.load(f)
+print("Projects in maintain json file: ", d['clean_project'])
+for proj in d['clean_project']:
+    now = arrow.now()
+    clean_time = now.to('local').shift(days=-3)
```

Comment:
Did you consider making the age an option? so that clean_time is something the user can adjust from the command line? e.g.

parser.add_argument("-a", "--age", default=3, help="The age, in days. If a project is older than this age it will be deleted.")

or something like that.

Reply:
I will make age as option. Thanks for your suggestion.